front/app/components/UI/ProjectFilter/index.tsx

```diff
@@ -68,18 +70,22 @@ const ProjectFilter = ({
   hideLabel = false,
   onProjectFilter,
   includeHiddenProjects = false,
+  id,
   ...boxProps
 }: Props & Omit<BoxProps, 'children'>) => {
   const localize = useLocalize();
   const { formatMessage } = useIntl();
+  const theme = useTheme();
+  const [searchValue, setSearchValue] = useState('');
+
   const { data: projects } = useProjects({
```

Comment:
Since this component is only used by admins, and this regular `useProjects` hook is awfully slow (on staging it takes like 5-10 seconds to load sometimes), the cherry on the cake would be to replace it by our new and fast `useInfiniteProjectsMiniAdmin` hook. That one can also handle sorting on the server side. But maybe out of scope.